apps/src/applab/designMode.js

```diff
@@ -288,7 +288,9 @@ designMode.updateProperty = function (element, name, value) {
       var height = parseInt(element.style.height, 10);
       element.style.backgroundSize = width + 'px ' + height + 'px';
 
-      var url = ICON_PREFIX_REGEX.test(value) ? assetPrefix.renderIconToString(value, element) : assetPrefix.fixPath(value);
+      var url = ICON_PREFIX_REGEX.test(value) ?
+        assetPrefix.renderIconToString(value, element) :
+        assetPrefix.fixPath(encodeURIComponent(value));
```

Comment:
No, this doesn't make sense to me.  For one thing, we don't do this for any other calls to `fixPath` - for another, fixPath is [already calling `encodeURIComponent` internally for absolute image URLs](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/assetManagement/assetPrefix.js#L31) _and_ also manually replacing spaces (I think for the image proxy's sake).  So if we are going to make this change, I suspect it should happen inside `fixPath`.

I'm also not sure this is the problem though.  When I load that project and inspect the background and the image, here's what I see:

![image](https://cloud.githubusercontent.com/assets/1615761/18720813/7aea591a-7fe2-11e6-8187-e739560844d0.png)

The background image URL is still "blockly/media/1x1.gif" (which, by the way, we've should replace with the [empty image dataUrl](https://github.com/code-dot-org/code-dot-org/blob/4c2ada7339d42d894d2be6e986016ddc00d98792/apps/src/gamelab/constants.js#L20) at some point) so it almost looks like we're hitting [this empty filename case](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/assetManagement/assetPrefix.js#L46) in `fixPath`.  What am I missing here?